api/src/main/java/org/apache/cloudstack/api/command/admin/router/ListRoutersCmd.java

```diff
@@ -132,6 +136,11 @@ public String getRole() {
         return Role.VIRTUAL_ROUTER.toString();
     }
 
+    public boolean shouldIncludeHealthCheckResults() {
+        return includeHealthCheckResults == null ? false : includeHealthCheckResults.booleanValue();
```

Comment:
This can be simplified to BooleanUtils.isTrue(includeHealthCheckResults) from the apache lang package

Reply:
Done.